app/database/group_item_store_compute_all_access.go

```diff
@@ -190,13 +171,18 @@ func (s *GroupItemStore) computeAllAccess() {
 	mustNotBeError(err)
 	defer func() { mustNotBeError(stmtMarkChildrenItems.Close()) }()
 
+	// ------------------------------------------------------------------------------------
+	// Here we execute the statements
+	// ------------------------------------------------------------------------------------
+	_, err = stmtDropTemporaryTable.Exec()
+	mustNotBeError(err)
+
 	hasChanges := true
 	for hasChanges {
-		_, err = stmtInsertMissingChildren.Exec()
-		mustNotBeError(err)
-		_, err = stmtInsertMissingPropagate.Exec()
+		_, err = stmtCreateTemporaryTable.Exec()
 		mustNotBeError(err)
-		_, err = stmtUpdatePropagateAccess.Exec()
+		mustNotBeError(s.Exec(queryInsertMissingChildren).Error())
+		_, err = stmtDropTemporaryTable.Exec()
 		mustNotBeError(err)
 		_, err = stmtMarkDoNotPropagate.Exec()
```

Comment:
I would expect `stmtMarkDoNotPropagate` to be run before creating the missing children (2 steps earlier), so that possibly many children are not created for "nothing". Do you agree?

Reply:
No, I do not agree here. Although we do not propagate access from custom chapters to their children, we still need to propagate access to custom chapters themselves.

Reply:
> No, I do not agree here. Although we do not propagate access from custom chapters to their children, we still need to propagate access to custom chapters themselves.

At least the 'database schema comments' doc says that. Nevertheless, AlogoreaPlatform even doesn't propagate access rights from to custom chapters themselves and we implemented that in the same way. Probably it's a bug. How should it work?

Reply:
Ah I see you did it at the end? 🤔
The change I was suggesting does not change anything to the logic, but just was not creating "empty" groups_items for nothing.  (except if I did a mistake)

Ah you have added `AND propagate_access = 'children'`. It looks like a good idea to make it correct, right.